docs/4.x/tagmanager/parameters.md

```diff
@@ -120,6 +120,28 @@ $field->transform = function ($value, Setting $setting) {
 }
 ```
 
+### Migrating tags to include new parameters
+
+When you add a new parameter to a tag template, all the existing tags referencing this template will be shown as changed after the next version is published. They will continue to do so until each one is manually updated in some way and another version is published. To avoid this, in Matomo version 4.12 forward, there is a helper class that can be used in migration scripts.
```

Comment:
@snake14 Does this look better to you ?
```suggestion
When you add a new parameter to a tag template, all the existing tags referencing this template will be shown as changed after the next version is published. They will continue to do so until each one is manually updated in some way and another version is published. To avoid this, there is a helper class that can be used in the migration script (Available from Matomo 4.12 and above versions).

Reply:
@AltamashShaikh Thank you. That does flow better than what I had. I made that change with a slight variation.